### PR TITLE
Add terminology tool tips to getting started pages

### PR DESCRIPTION
**Pull request by dannytsang**
## Proposed change
Add terminology tool tips to getting started pages.

Removed HTML image syntax with Markdown.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dannytsang** [comment]
@frenck I went conservative and used the first occurrence of the word and added the tool tip rather than replacing all of the words on a page. If the preference is to replace all words on a page then let me know.

Either way, I feel it might be worth mentioning the stance on this on https://developers.home-assistant.io/docs/documenting/standards/#glossary--terminology-tooltips

File: source/getting-started/automation.markdown [hunk 1]
```diff
@@ -3,43 +3,31 @@ title: "Automating Home Assistant"
 description: "A quick intro on getting your first automation going."
 ---
 
-Once your devices are set up, it's time to put the cherry on the pie: automation. In this guide we're going to create a simple automation rule to turn on the lights when the sun sets. Of course, this assumes that you have set up an integration that provides a light at this point.
+Once your {% term devices %} are set up, it's time to put the cherry on the pie: {% term automation %}. In this guide we're going to create a simple automation rule to turn on the lights when the sun sets. Of course, this assumes that you have set up an integration that provides a light at this point.
 
-In the user interface, click **Settings** in the sidebar, then click **Automations & Scenes**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
+In the user interface in the side bar, click **{% my automations title="Settings -> Automations & Scenes" %}**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```
Comment (frenck) [inline source/getting-started/automation.markdown]: - The sidebar is a single thing (not the bar on the side, but the sidebar).
- Replace `->` by `>`, which we are trying to make a standard.

```suggestion
In the user interface in the sidebar, click **{% my automations title="Settings > Automations & Scenes" %}**. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small suggestion/comment.

> I went conservative and used the first occurrence of the word and added the tool tip rather than replacing all of the words on a page. If the preference is to replace all words on a page then let me know.

I think conservative is the way to go. It is meant to provide context to words that haven't been introduced yet (similar to how one would process abbreviations)?

@c0ffeeca7 What do you think?

**Comment by c0ffeeca7** [comment]
> Small suggestion/comment.
> 
> > I went conservative and used the first occurrence of the word and added the tool tip rather than replacing all of the words on a page. If the preference is to replace all words on a page then let me know.
> 
> I think conservative is the way to go. It is meant to provide context to words that haven't been introduced yet (similar to how one would process abbreviations)?
> 
> @c0ffeeca7 What do you think?

Agreed. I am unsure about adding a rule to the guides about only adding it to the first occurrence, though. I think one should feel free to add them wherever it seems necessary. If people go berserk with it we can still add it.

File: source/getting-started/automation.markdown (hunk 1 above)
Comment (dannytsang) [reply source/getting-started/automation.markdown]: Got it. In terms of the changes like -> to >, is there a place where I can follow these discussions or latest conventions?

Comment (c0ffeeca7) [reply source/getting-started/automation.markdown]: There seem to be different styles in use: >>, -> and >.
we are trying to follow the Microsoft Style Guide, which uses >.
https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

**Comment by frenck** [comment]
@dannytsang This PR looks ready to go? (Besides it being draft and a merge conflict)

**Review by c0ffeeca7 — APPROVED** [review]
love it, @dannytsang :tada:

**Review by frenck — APPROVED** [review]
Nice! 🎉

**Pull request merged**